### DISCUSSION TITLE
Quick Reblog: Blog icon next to blog selector

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#481] [Suggestion Box]
The CSS might be a bit of a nightmare, and honestly it might not actually look any good, but I had the idea to add the icon of the selected blog to the side of the blog selector in the quick reblog popup. This is nice if, like me, you have multiple blogs with visually similar names (a poor decision on my part, perhaps, but oh well.)

I don't plan to try implementing this at the moment, but I don't think it would be difficult. The hard part, I would imagine, would be making the layout look good; there's not a lot of space :D

**Comment by AprilSylph** [comment]
I tried this once, can't remember what the problem was with it. Maybe I should have another go at it.
